commands/import.go

```diff
@@ -36,33 +35,40 @@ func init() {
 // doImport runs the importer
 func doImport(c *cli.Context) error {
 	res := database.InitResources(c.String("config"))
-	importDir := c.String("import-dir")
-	databaseName := c.String("database")
+	importDir := c.Args().Get(0)
+	targetDatabase := c.Args().Get(1)
 	threads := util.Max(c.Int("threads")/2, 1)
 
-	//one flag was set
-	if importDir != "" && databaseName == "" || importDir == "" && databaseName != "" {
-		return cli.NewExitError(
-			"Import failed.\nUse 'rita import' to import the directories "+
-				"specified in the config file or 'rita import -i [import-dir] -d [database-name]' "+
-				"to import bro logs from a given directory.", -1)
+	//check if one argument is set but not the other
+	if importDir != "" && targetDatabase == "" ||
+		importDir == "" && targetDatabase != "" {
+		return cli.NewExitError("Both <directory to import> and <database prefix> are required to override the config file.", -1)
 	}
 
-	//both flags were set
-	if importDir != "" && databaseName != "" {
-		res.Config.S.Bro.LogPath = importDir
-		res.Config.S.Bro.DBPrefix = ""
-		//Clear out the directory map and set the default database
-		res.Config.S.Bro.DirectoryMap = make(map[string]string)
-		res.Config.S.Bro.DefaultDatabase = databaseName
+	//check if the user overrode the config file
+	if importDir != "" && targetDatabase != "" {
+		//expand relative path
+		//nolint: vetshadow
+		importDir, err := filepath.Abs(importDir)
+		if err != nil {
+			return cli.NewExitError(err.Error(), -1)
+		}
+
+		//Remove tailing / when applicable
```

Comment:
I think this should be outside the `if` block. This currently only gets executed when the config file is overridden.

Reply:
Nvm, I found where this is modifying the config value in `config/static.go`. The bug is there.